modAionImpl/src/org/aion/zero/impl/sync/statistics/TopSeedsStatsTracker.java

```diff
@@ -0,0 +1,161 @@
+package org.aion.zero.impl.sync.statistics;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.stream.Collectors;
+import org.apache.commons.collections4.map.LRUMap;
+
+public class TopSeedsStatsTracker {
+
+    private final Map<String, Integer> receivedByPeer;
```

Comment:
Should be refactored using an enum similarly to the `ResponseStatsTracker`

Reply:
refactored using new enum